### DISCUSSION TITLE
Is it possible to plot using DimPlot directly?

### DISCUSSION THREAD

**Discussion by Weiming113** [GreenleafLab/ArchR#1848] [Feature Requests]
Hello,

ArchR is a really excellent tool. I know we can plot using plotEmbedding() function. But when I am trying to use the DimPlot() function, it seems the ArchRProject doesn't match. I am asking if it is possible to make the project of ArchR is able to be used by DimPlot directly? Or could you please give some guidelines how to plot using DimPlot on ArchR project.

Thanks

**Comment by rcorces** [comment]
ArchR is not part of the Seurat ecosystem and it should never be assumed that a Seurat function would work on an ArchR object. This is not supported.

**Reply by Weiming113** [reply]
Thanks for you quick reply. 
As a new user, I was failing to change the cell cluster labels from like 1,2,3,4... to the real name of the cell type using plotEmbedding() function.  Could you please help how to do that?

**Reply by rcorces** [reply]
I would encourage you to read the manual, with particular attention to how to update `cellColData`
